Data/Scripts/GardenConquest/Messaging/ResponseProcessor.cs

```diff
@@ -66,14 +66,32 @@ public class ResponseProcessor {
 				// Deserialize the message
 				BaseResponse msg = BaseResponse.messageFromBytes(buffer);
 
+				// Check for missing important parts
+				if (msg.Destination == null || msg.DestType == null) {
```

Comment:
DestType is an enum and will never be null.
It is also possible for Destination to be null in the event of an EVERYONE message.